Assets/MixedRealityToolkit.Examples/Experimental/Demos/UX/ObjectManipulator/Scripts/ReturnToBounds.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ReturnToBounds : MonoBehaviour
+{
+    [SerializeField]
+    Transform frontBounds = null;
```

Comment:
Missing [access modifiers](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#access-modifiers)

Reply:
Is this required even for demo scripts?

Reply:
I've been trying to treat all code that goes in the repo with the same quality, regardless of the module it ends up in

Reply:
Cool, sounds good to me